### PR TITLE
Mafia app branch

### PR DESCRIPTION
**Pull request by joyeeta26**

### CODE REVIEW

File: src/GameLayout/GameLayout.js [hunk 1]
```diff
@@ -1,14 +1,63 @@
...
 
-function GameLayout() {
+function GameLayout(props) {
+  const roundState = "Night";                                 //day or night
+  const primaryMafiaName="Kriti";                     
+  const yourName="Joyeeta";
+  const previousKillRole = "Mafia";                      //Villager of Mafia
+
+  const Winners = [
+    {name: "Sharon", role: "Villager"},
```
Comment (RonMathews) [inline src/GameLayout/GameLayout.js]: move all these

File: src/LobbyLayout/GameSettings.js [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
 import { Button, TextField } from '@fluentui/react';
-import { containerStyles, invitationCodeTextStyles, invitationCodeStyles, getTextStyles, getButtonStyles } from "./GameSettingsStyles";
+import { containerStyles, invitationCodeTextStyles, invitationCodeStyles, getTextStyles, getButtonStyles, waitingMsgTextStyles, conferenceLinkTextStyles, conferenceLinkStyles} from "./GameSettingsStyles";
 
-function GameSettings() {
-  const [state, dispatch] = useGlobalState();
-  const enoughPlayers = true; // todo read this from some global state post roster syncing
+function GameSettings(props) {
+  const enoughPlayers = props.noOfPlayers > 6; // todo read this from some global state post roster syncing
   const gameCode = "HDJF"; // todo read this from state
```
Comment (RonMathews) [inline src/LobbyLayout/GameSettings.js]: take in from props

File: src/GameLayout/GameLayout.js (hunk 1 above)
Comment (joyeeta26) [reply src/GameLayout/GameLayout.js]: Done

File: src/LobbyLayout/GameSettings.js (hunk 2 above)
Comment (joyeeta26) [reply src/LobbyLayout/GameSettings.js]: Done

**Review by RonMathews — APPROVED** [review]

**Pull request merged**